src/auto-evo/AutoEvoRun.cs

```diff
@@ -123,17 +126,18 @@ public string Status
             if (Finished)
                 return TranslationServer.Translate("FINISHED_DOT");
 
-            if (!Running)
-                return TranslationServer.Translate("NOT_RUNNING_DOT");
+            if (!started)
+                return TranslationServer.Translate("NOT_STARTED_DOT");
 
             int total = totalSteps;
 
             if (total > 0)
             {
                 var percentage = CompletionFraction * 100;
 
-                // {0:F1}% done. {1:n0}/{2:n0} steps.
-                return TranslationServer.Translate("AUTO-EVO_STEPS_DONE").FormatSafe(percentage, CompleteSteps, total);
+                // {0:F1}% done. {1:n0}/{2:n0} steps. [Paused.]
+                return TranslationServer.Translate("AUTO-EVO_STEPS_DONE").FormatSafe(percentage, CompleteSteps, total)
+                    + (Running ? string.Empty : " " + TranslationServer.Translate("PAUSED_DOT"));
```

Comment:
I'd rename "PAUSED_DOT" key to "OPERATION_PAUSED_TAG" , and change the text to "[Paused]"

Reply:
See my overall review for an update on this.